libcodechecker/cmd/cmd_line_client.py

```diff
@@ -465,6 +474,69 @@ def printResult(getterFn, baseid, newid, suppr, output_format):
                     args.output_format)
 
 
+def handle_suppress(args):
+    def update_suppression_comment(run_id, report_id, comment):
+        client.unSuppressBug([run_id], report_id)
+        client.suppressBug([run_id], report_id, comment)
+
+    def bug_hash_filter(bug_id, filepath):
+        filepath = '%' + filepath
+        return [
+            codeCheckerDBAccess.ttypes.ReportFilter(bugHash=bug_id,
+                                                    suppressed=True,
+                                                    filepath=filepath),
+            codeCheckerDBAccess.ttypes.ReportFilter(bugHash=bug_id,
+                                                    suppressed=False,
+                                                    filepath=filepath)]
+
+    already_suppressed = 'Bug {} in file {} already suppressed. Use --force!'
+    limit = codeCheckerDBAccess.constants.MAX_QUERY_SIZE
+
+    client = setupClient(args.host, args.port, '/')
+
+    run_info = check_run_names(client, [args.name])
+    run_id, run_date = run_info.get(args.name)
+
+    if args.output:
+        with open(args.output, 'w') as out:
```

Comment:
Writing and reading the suppress file should be done only through the [https://github.com/Ericsson/codechecker/blob/master/libcodechecker/suppress_file_handler.py](suppress_file_handler.py) module. That module handles the reading and writing of the suppress file (the format is specified there).